wolfssl/wolfcrypt/error-crypt.h

```diff
@@ -65,11 +65,17 @@ enum {
     MP_ZERO_E          = -121,  /* got a mp zero result, not expected */
 
     AES_EAX_AUTH_E     = -122, /* AES-EAX Authentication check failure */
+    KEY_EXHAUSETED_E   = -123, /* No longer usable for operation. */
```

Comment:
Same typo.

Also, is this error code intended for stateful HBS methods only, or just in general for a key with a finite signature space?

Reply:
Fixed